docs/topics/inline-functions.md

```diff
@@ -54,7 +54,9 @@ however, can be manipulated in any way you like, including being stored in field
 >
 {style="note"}
 
-## Non-local returns
+## Non-local jump expressions
```

Comment:
The documentation on non-local break/continue/return is duplicated in `inline-functions.md` ("Inline functions") and `returs.md` ("Returns and jumps")

I think, it'd be better to place the docs only to one of the pages. and another page could just have a link to the first one.

But if you think that duplication is not a problem, you can keep it "as is". It's up to you.

Reply:
To my mind, "non-local jumps" is more of a feature of `inline` functions. And the `returns.md` could be renamed to smth like `jump-labels.md` and be minimized to only talk about labels

Reply:
In my mind, non-local feature was as important as the labels, so I created an equal structure for both of them in returns.md

But you're right, if it's just a specific feature of inline functions, we can minimize it in returns.md down to just a link to the main Inline functions page with detailed info and avoid duplication